core/server/master/src/main/java/alluxio/master/file/meta/InodeFile.java

```diff
@@ -103,6 +104,16 @@ public void reset() {
     mCacheable = false;
   }
 
+  @Override
+  public DefaultAccessControlList getDefaultACL() throws UnsupportedOperationException {
+    throw new UnsupportedOperationException("File does not have default ACL");
+  }
+
+  @Override
+  public void setDefaultACL(DefaultAccessControlList acl) throws UnsupportedOperationException {
+    throw new UnsupportedOperationException("File does not have default ACL");
```

Comment:
NIT: can we distinguish between this message and the one above? That will help any debugging.

Reply:
done